src/types.rs

```diff
@@ -36,7 +36,7 @@ impl Token {
         Token {
             inner: Arc::new(InnerToken {
                 access_token,
-                expires_at: None,
+                expires_at: OffsetDateTime::now_utc() + Duration::seconds(3600),
```

Comment:
Hmm, I don't like that `Token::from_string()` implicitly comes up with the `Duration::seconds(3600)`. I think it should take a `Duration` argument instead, letting the caller deal with that, which properly forces the only caller in `gcloud_authorized_user` -- I think we'll want to add a comment there about the context in which we're putting this (that is -- this seems to be the default in practice but we don't have any references that it'll always be like this).